tests/basic_one_way_send.go

```diff
@@ -2,30 +2,33 @@ package tests
 
 import (
 	"context"
+	"encoding/hex"
 	"math/big"
 
 	"github.com/ava-labs/subnet-evm/interfaces"
 	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/TeleporterMessenger"
+	"github.com/ava-labs/teleporter/tests/network"
 	"github.com/ava-labs/teleporter/tests/utils"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/log"
 	. "github.com/onsi/gomega"
 )
 
-// Ginkgo describe node that acts as a container for the teleporter e2e tests. This test suite
-// will run through the following steps in order:
-// 1. Send a transaction to the Teleporter contract on Subnet A
-// 2. Aggregate signatures and send the Warp message to Subnet B
-// 3. Verify receipt of the message on Subnet B
-func BasicOneWaySend() {
+func BasicOneWaySendGinkgo() {
```

Comment:
We have wrapper function for `BasicOneWaySend`. Is it because we do not allow to pass parameter when calling the test in [e2e_test.go](https://github.com/ava-labs/teleporter/pull/93/files#diff-311b8f116ec381569dfecd92d10675b9ff9acffcbc86d66ca4bc0aca79b6e991R45)?

Reply:
Yes, exactly. The different `network.Network` implementations point to the local network used in the E2E tests, as well as an example Fuji network that can be run against a live Fuji network.